src/constants/index.js

```diff
@@ -86,9 +86,11 @@ export const releaseMapper = {
   'rhel-86': 'Red Hat Enterprise Linux (RHEL) 8.6',
   'rhel-85': 'Red Hat Enterprise Linux (RHEL) 8.5',
   'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
+  'rhel-81': 'Red Hat Enterprise Linux (RHEL) 8.1',
```

Comment:
Shouldn't it be ?
```suggestion
  'rhel-81': 'Red Hat Enterprise Linux (RHEL) 8.10',
```

Reply:
We can discuss deprecating minor versions, but we should keep 8.1 and add 8.10 here and in the supportedReleases below.

Reply:
The key must be: `rhel-8.10` (yeah it is different).